*** ./smali/android/telephony/SmsMessage.smali	2016-01-12 13:10:53.602171007 +0700
--- /home/ngxson/nicki/miui/temp/new_smali/telephony-common.jar.out/./smali/android/telephony/SmsMessage.smali	2016-01-12 13:13:15.819025655 +0700
***************
*** 829,834 ****
--- 829,847 ----
      return-object v0
  .end method
  
+ .method public getEncodingType()I
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Landroid/telephony/SmsMessage;->mWrappedSmsMessage:Lcom/android/internal/telephony/SmsMessageBase;
+ 
+     invoke-virtual {v0}, Lcom/android/internal/telephony/SmsMessageBase;->getEncodingType()I
+ 
+     move-result v0
+ 
+     return v0
+ .end method
+ 
  .method public getIndexOnIcc()I
      .locals 1
  
