*** ./smali/android/preference/RingtonePreference.smali	2016-01-12 13:11:35.960980605 +0700
--- /home/ngxson/nicki/miui/temp/new_smali/framework.jar.out/./smali/android/preference/RingtonePreference.smali	2016-01-12 13:13:58.557645249 +0700
***************
*** 311,316 ****
--- 311,322 ----
  
      invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->putExtra(Ljava/lang/String;Ljava/lang/CharSequence;)Landroid/content/Intent;
  
+     const-string v0, "com.android.thememanager"
+ 
+     const-string v1, "com.android.thememanager.activity.ThemeTabActivity"
+ 
+     invoke-virtual {p1, v0, v1}, Landroid/content/Intent;->setClassName(Ljava/lang/String;Ljava/lang/String;)Landroid/content/Intent;
+ 
      return-void
  .end method
  
