*** ./smali/android/app/FragmentManagerImpl$6.smali	2016-01-12 13:12:10.443730277 +0700
--- /home/ngxson/nicki/miui/temp/new_smali/framework.jar.out/./smali/android/app/FragmentManagerImpl$6.smali	2016-01-12 13:14:34.867480904 +0700
***************
*** 49,54 ****
--- 49,60 ----
  
      iget-object v0, p0, Landroid/app/FragmentManagerImpl$6;->val$finalFragment:Landroid/app/Fragment;
  
+     iget-boolean v0, v0, Landroid/app/Fragment;->mHidden:Z
+ 
+     if-eqz v0, :cond_0
+ 
+     iget-object v0, p0, Landroid/app/FragmentManagerImpl$6;->val$finalFragment:Landroid/app/Fragment;
+ 
      iget-object v0, v0, Landroid/app/Fragment;->mView:Landroid/view/View;
  
      const/16 v1, 0x8
