*** ./smali/android/widget/Toast$TN.smali	2016-01-12 13:12:03.910998339 +0700
--- /home/ngxson/nicki/miui/temp/new_smali/framework.jar.out/./smali/android/widget/Toast$TN.smali	2016-01-12 13:14:28.182824968 +0700
***************
*** 168,173 ****
--- 168,182 ----
  
  
  # virtual methods
+ .method getParams()Landroid/view/WindowManager$LayoutParams;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Landroid/widget/Toast$TN;->mParams:Landroid/view/WindowManager$LayoutParams;
+ 
+     return-object v0
+ .end method
+ 
  .method public handleHide()V
      .locals 2
  
