*** ./smali/com/android/server/LightsService.smali	2016-01-12 13:10:33.884035194 +0700
--- /home/ngxson/nicki/miui/temp/new_smali/services.jar.out/./smali/com/android/server/LightsService.smali	2016-01-12 13:12:55.385247850 +0700
***************
*** 207,209 ****
--- 207,222 ----
  
      return-object v0
  .end method
+ 
+ .method public setLight(ILcom/android/server/LightsService$Light;)V
+     .locals 1
+     .param p1, "index"    # I
+     .param p2, "light"    # Lcom/android/server/LightsService$Light;
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/server/LightsService;->mLights:[Lcom/android/server/LightsService$Light;
+ 
+     aput-object p2, v0, p1
+ 
+     return-void
+ .end method
